### PR TITLE
Add node-monero-rpc library

### PR DESCRIPTION
**Pull request by ExcitableAardvark**

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@
       url: https://github.com/PsychicCat/monero-php
     - name: MoneroPy (Python)
       url: https://github.com/bigreddmachine/MoneroPy
+    - name: node-monero-rpc (Node.js)
+      url: https://github.com/ExcitableAardvark/node-monero-rpc#readme
```
Comment (leonklingele) [inline _data/merchants.yml]: I'd remove `#readme` for consistency.

**Comment by el00ruobuob** [comment]
@ExcitableAardvark could you apply @leonklingele recommandation and rebase on current master?

**Comment by ExcitableAardvark** [comment]
@el00ruobuob Sorry for the delay, updated.

**Comment by ExcitableAardvark** [comment]
@erciccione Sorry, fixed

**Pull request closed without merging**